testsuite/src/test_libblis.c

```diff
@@ -860,12 +856,15 @@ void libblis_test_output_params_struct( FILE* os, test_params_t* params )
 	bli_rntm_init_from_global( &trsm_l );
 	bli_rntm_init_from_global( &trsm_r );
 
+    /*
+	dim_t  m = 1000, n = 1000, k = 1000;
 	bli_rntm_set_ways_for_op( BLIS_GEMM, BLIS_LEFT,  m, n, k, &gemm );
 	bli_rntm_set_ways_for_op( BLIS_HERK, BLIS_LEFT,  m, n, k, &herk );
 	bli_rntm_set_ways_for_op( BLIS_TRMM, BLIS_LEFT,  m, n, k, &trmm_l );
 	bli_rntm_set_ways_for_op( BLIS_TRMM, BLIS_RIGHT, m, n, k, &trmm_r );
 	bli_rntm_set_ways_for_op( BLIS_TRSM, BLIS_LEFT,  m, n, k, &trsm_l );
 	bli_rntm_set_ways_for_op( BLIS_TRSM, BLIS_RIGHT, m, n, k, &trsm_r );
+    */
```

Comment:
@devinamatthews What was the thinking behind removing this portion of the output from the testsuite info header?